_posts/2017-02-09-CoC.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: Our Code of Conduct
+---
+
+
+Dear MDAnalysis Community,
+
```

Comment:
I would start with a happy note. Just a short sentence like: "We feel the MDAnalysis community is a happy one, and we hope all member feel the same. As the community grows we want it to stay as welcoming. In that spirit, MDAnalysis has adopted..."